client/src/components/Tickets/TicketModal.js

```diff
@@ -40,7 +40,7 @@ const Attachments = ({ attachments }) => {
       Attachments:
       <ul>
         {attachments.map(attachmentName => (
-          <li> {attachmentName[1]} </li>
+          <li key={attachmentName[0]}>{attachmentName[1]} </li>
```

Comment:
Do you know that the first element in the array is necessarily unique for an attachment? I'm not actually sure what that represents. @nathanfranklin ?